modules/core/include/opencv2/core/utility.hpp

```diff
@@ -56,6 +56,10 @@
 #include "opencv2/core.hpp"
 #include <ostream>
 
+#if __cplusplus >= 201103L
```

Comment:
May be we should replace separate features detection with common `CV_CXX11` in `cvdef.h`